numpy-stubs/__init__.pyi

```diff
@@ -130,16 +132,18 @@ class _flagsobj:
     def __getitem__(self, key: str) -> bool: ...
     def __setitem__(self, key: str, value: bool) -> None: ...
 
-class flatiter:
+_FlatIterSelf = TypeVar("_FlatIterSelf", bound=flatiter)
+
+class flatiter(Generic[_ArraySelf], Iterator[generic]):
```

Comment:
I don't think you need the inheritance from `Iterator` here; the `__iter__` and `__next__` defined below should cover it.

Reply:
I've removed it as of https://github.com/numpy/numpy-stubs/pull/78/commits/b5341a592d414ad26486f2ecec2a6a3e030a5e44.